jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpCompliance.java

```diff
@@ -222,7 +222,7 @@ public static HttpCompliance valueOf(String name)
      * <dl>
      * <dt>0</dt><dd>No {@link Violation}s</dd>
      * <dt>*</dt><dd>All {@link Violation}s</dd>
-     * <dt>RFC2616</dt><dd>The set of {@link Violation}s application to https://tools.ietf.org/html/rfc2616,
+     * <dt>RFC2616</dt><dd>The set of {@link Violation}s application to <a href="https://tools.ietf.org/html/rfc2616">...</a>,
```

Comment:
I think this javadoc should be reverted. Or the link should be given a text other than `...` and other links should be edited too.